vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -641,8 +641,10 @@ void ReactInstanceWin::InitializeBridgeless() noexcept {
               auto timerManager = std::make_shared<facebook::react::TimerManager>(std::move(timerRegistry));
               timerRegistryRaw->setTimerManager(timerManager);
 
```

Comment:
@acoates-ms there was a change upstream that changed a lot of the `JsErrorHandler` code upstream - including the `OnJsError()` method that we use when creating a `ReactInstance`. Do these changes look good to accommodate the upstream changes?

Reply:
yup looks fine